src/idl_gen_python.cpp

```diff
@@ -141,9 +141,18 @@ static void GetScalarFieldOfTable(const StructDef &struct_def,
   code += MakeCamel(field.name);
   code += "(self):";
   code += OffsetPrefix(field);
-  code += Indent + Indent + Indent + "return " + getter;
-  code += "o + self._tab.Pos)\n";
-  code += Indent + Indent + "return " + field.value.constant + "\n\n";
+  getter += "o + self._tab.Pos)";
+
+  bool isBool = field.value.type.base_type == BASE_TYPE_BOOL;
```

Comment:
please use code style of surrounding code, e.g. `is_bool`, and no random blank lines.